lib/rest-adapter.js

```diff
@@ -403,6 +411,11 @@ RestAdapter.prototype._registerMethodRouteHandlers = function(router,
     // Express 4.x only supports delete
     verb = 'delete';
   }
+
+  if (fastRouter.canRegister(route, handler)) {
+    handler.methodName = sharedMethod.name;
+    fastRouter[verb](route, handler);
```

Comment:
@bajtos : I will set it to `sharedMethod.stringName` thus it would be clear if it's a prototype method or a static method, and as you recommended, will use `displayName` as:
`handler.displayName = sharedMethod.stringName`
Is that fine?